### ISSUE TITLE
A new icon (mainly for Firefox OS)

### ISSUE THREAD

**Issue by FabianOvrWrt** [OthelloVenturesLtd/FeedSpider2#1]
Some friends have been using this app, but they always tell me that the icon doesn't look good on Firefox OS. I have been "redesigning" some of the icons for the apps that I use [https://github.com/loqui/im/pull/569 or https://github.com/mozillahispano/openwapp/issues/13#issuecomment-62852128].

The new logo is based of a jumping spider, because they're cute and not scary at all (for most people).
I would like to hear your opinion about my icon to fix problems that you might have with it (maybe making a squared version for other platforms or using different colors).

As I stated on the OpenWapp issue above, I can make any size of this because I work with vector graphics.

![284](https://cloud.githubusercontent.com/assets/6712155/8612683/337e9250-269a-11e5-852a-e0360c0166ea.png)

**Comment by Aressel** [comment]
Sorry for taking so long to respond to this. I have been absolutely swamped for the past two and a half weeks or so. I like it! If you can send me a vector version, and let me know how you'd like to be credited, I'll make it the new logo.

**Comment by FabianOvrWrt** [comment]
Yes, you can credit me in any way you want. Where do I send the file?

**Comment by OthelloVentures** [comment]
Please send to feedspider@feedspider.net. For the credit, what name and url or email address would you like me to use?

From:  Fabián PS
Reply-To:  OthelloVenturesLtd/FeedSpider2
Date:  Tuesday, July 28, 2015 at 5:30 PM
To:  OthelloVenturesLtd/FeedSpider2
Subject:  Re: [FeedSpider2] A new icon (mainly for Firefox OS) (#1)

Yes, you can credit me in any way you want. Where do I send the file?
